### PR TITLE
deploy web app to azure blob storage

### PR DESCRIPTION
**Pull request by farhadbahrehmandhenry**
- issue #133

- deploy APP (UI) to azure blob storage

# how to use =>
- after merging a PR this action gets triggered.
- you can follow the action's step in Github/Actions/deploy-ui
- if the action completes the job, you will have your latest changes deployed in this [URL](https://nscstrdevusw2thucommon.z5.web.core.windows.net/)
- please note you should wait until this PR gets merged first.

# spent time =>
about 6 hours

### CODE REVIEW

**Review by theKunte — APPROVED** [review]

File: ui/react-ui/src/components/routes/userSingleTask/UserSingleTask.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ let UserSingleTask = () => {
   console.log(userId, taskId)
 
   useEffect(() => {
-    fetch(`https://nsc-functionsapp-team1.azurewebsites.net/api/users/${userId}/tasks/${taskId}`)
+    fetch(`https://nsc-fun-dev-usw2-thursday.azurewebsites.net/api/users/${userId}/tasks/${taskId}`)
```
Comment (jak-ashuraliev) [inline ui/react-ui/src/components/routes/userSingleTask/UserSingleTask.js:13]: Updated!

File: ui/react-ui/src/components/routes/userTasks/UserTasks.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ var UserTasks = () => {
   useEffect(() => {
     setUserTasks(['loading']);
 
-    fetch(`https://nsc-functionsapp-team1.azurewebsites.net/api/users/${userId}/tasks?`)
+    fetch(`https://nsc-fun-dev-usw2-thursday.azurewebsites.net/api/users/${userId}/tasks?`)
```
Comment (jak-ashuraliev) [inline ui/react-ui/src/components/routes/userTasks/UserTasks.js:14]: Updated!

File: ui/react-ui/src/components/routes/userid/userid.js [hunk 3]
```diff
@@ -10,9 +10,15 @@ var User = () => {
   var {userId} = useParams();
 
   useEffect(() => {
-    fetch(`https://nsc-functionsapp-team1.azurewebsites.net/api/users/${userId}?`)
+    fetch(`https://nsc-fun-dev-usw2-thursday.azurewebsites.net/api/users/${userId}?`)
```
Comment (jak-ashuraliev) [inline ui/react-ui/src/components/routes/userid/userid.js:13]: Updated!

File: ui/react-ui/src/components/routes/users/Users.js [hunk 4]
```diff
@@ -11,7 +11,7 @@ export default class Users extends Component {
 	};
 
 	async componentDidMount() {
-		const url = 'https://nsc-functionsapp-team1.azurewebsites.net/api/users';
+		const url = 'https://nsc-fun-dev-usw2-thursday.azurewebsites.net/api/users';
```
Comment (jak-ashuraliev) [inline ui/react-ui/src/components/routes/users/Users.js:14]: And thanks for updating users as well.

File: ui/react-ui/src/components/app/App.css [hunk 5]
```diff
@@ -1,6 +1,4 @@
 .App {
```
Comment (jak-ashuraliev) [inline ui/react-ui/src/components/app/App.css:1]: Good clean up.

**Review by jak-ashuraliev — APPROVED** [review]
Overall, good updates, cleanup, and deployment work like a charm, brother. You should apply to AWS.

**Pull request merged**